apply/run.go

```diff
@@ -61,6 +63,47 @@ func IsIPList(args string) bool {
 	return true
 }
 
+func PreProcessIPList(joinArgs *common.RunArgs) {
```

Comment:
Provide some test cases for this function.
Attention about the edge cases like: `x.x.x.x-` x.x.x.x-x.x.x.x-x.x.x.x` ...